test/common/network/connection_balancer_impl_test.cc

```diff
@@ -0,0 +1,61 @@
+#include "envoy/config/listener/v3/listener.pb.h"
```

Comment:
Sorry I meant to add an integration test. I think there are existing integration tests for connection balancing so you can model off of that.

/wait